### PR TITLE
Updates kinematics information for the igus

### PR DESCRIPTION
**Pull request by Bidski**
- Corrects knee offset value so that hip pitch, knee, and ankle pitches all align vertically in zombie.
- Provides accurate measurements for limb lengths and distances between servos
- Adds an extra offset from hip roll to hip pitch in the forward kinematics calculations (does something need to happen in IK too?)
- Renames arm kinematics in config file and adds comments to for each parameter
- Removes darwin configs and makes the igus config the default one

### CODE REVIEW

File: module/platform/darwin/HardwareIO/data/config/HardwareIO.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 battery:
```
Comment (JakeFountain) [inline module/platform/darwin/HardwareIO/data/config/HardwareIO.yaml]: I dont know about these values

File: module/motion/KinematicsConfiguration/src/KinematicsConfiguration.cpp [hunk 2]
```diff
@@ -114,37 +114,25 @@ namespace motion {
...
-        model.arm.SHOULDER_LENGTH            = objShoulder["length"].as<float>();
-        model.arm.SHOULDER_WIDTH             = objShoulder["width"].as<float>();
-        model.arm.SHOULDER_HEIGHT            = objShoulder["height"].as<float>();
-        model.arm.UPPER_ARM_LENGTH           = objUpperArm["length"].as<float>();
-        model.arm.UPPER_ARM_Y_OFFSET         = objUpperArm["offset"].as<arma::vec2>()[0];
-        model.arm.UPPER_ARM_X_OFFSET         = objUpperArm["offset"].as<arma::vec2>()[1];
-        model.arm.LOWER_ARM_LENGTH           = objLowerArm["length"].as<float>();
-        model.arm.LOWER_ARM_Y_OFFSET         = objLowerArm["offset"].as<arma::vec2>()[0];
-        model.arm.LOWER_ARM_Z_OFFSET         = objLowerArm["offset"].as<arma::vec2>()[1];
+        arma::vec3 shoulder         = objArm["shoulder"].as<arma::vec3>();
```
Comment (JakeFountain) [inline module/motion/KinematicsConfiguration/src/KinematicsConfiguration.cpp]: LGTM

File: shared/utility/motion/ForwardKinematics.h [hunk 3]
```diff
@@ -157,6 +157,8 @@ namespace motion {
                 return positions;
             }
 
+            // Translate to hip pitch joint
+            runningTransform = runningTransform.translateZ(model.leg.HIP_PITCH_OFFSET);
```
Comment (JakeFountain) [inline shared/utility/motion/ForwardKinematics.h]: This needs to happen before the hip roll transform is set.

File: module/platform/darwin/HardwareIO/data/config/HardwareIO.yaml (hunk 1 above)
Comment (Bidski) [reply module/platform/darwin/HardwareIO/data/config/HardwareIO.yaml]: Most of these changes relate to the old default configuration file being for the darwin. The only values of interest here are the knee offset which was updated from `pi / 8` to `0.45` which we determined from `scripttuner` and zombie

**Comment by tayloryoung6396** [comment]
This should be merged

**Pull request closed without merging**